### PR TITLE
Initial webserver endpoint

### PR DESCRIPTION
**Pull request by jFlamer**
It starts webserver with only one endpoint /homep.

### CODE REVIEW

File: nsc_start.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+homep = {"id" : "Welcome to colorfull nonogram solver!"}
+api = Flask(__name__)
+
+@api.route('/homep', methods=['GET'])
+def get_homep():
+  return json.dumps(homep)
+
+if __name__ == '__main__':
+    api.run()
```
Comment (qualv13) [inline nsc_start.py:11]: Looks fine to me. @Frendzlu can say his opinion

**Review by qualv13 — CHANGES_REQUESTED** [review]
change _homep_ name and is ok for me

File: nsc_start.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from flask import Flask, json
+
+homep = {"id" : "Welcome to colorfull nonogram solver!"}
+api = Flask(__name__)
+
+@api.route('/homep', methods=['GET'])
```
Comment (Frendzlu) [inline nsc_start.py:6]: it should actually just be "/", as it is the default starting endpoint

File: nsc_start.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from flask import Flask, json
+
+homep = {"id" : "Welcome to colorfull nonogram solver!"}
```
[the remark this answers is not in the document]
Comment (Frendzlu) [reply nsc_start.py:3]: agree, homeData would perhaps be good

File: openapi.yaml [hunk 4]
```diff
@@ -0,0 +1,221 @@
+openapi: 3.0.3
+info:
+  title: Swagger Petstore - OpenAPI 3.0
+  version: 1.0.11
```
Comment (Frendzlu) [inline openapi.yaml:4]: change version to 0.0.1

File: openapi.yaml [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+externalDocs:
+  description: Find out more about Swagger
```
Comment (Frendzlu) [inline openapi.yaml:6]: Change to `Find out more about our project`
and make it point to google docs, or our wiki page, or the webapp.
Actually, perhaps just the webapp repository.

File: openapi.yaml [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+  url: http://swagger.io
+servers:
```
Comment (Frendzlu) [inline openapi.yaml:8]: delete this section for now

File: openapi.yaml [hunk 7]
```diff
@@ -0,0 +1,221 @@
+openapi: 3.0.3
+info:
+  title: Swagger Petstore - OpenAPI 3.0
```
Comment (Frendzlu) [inline openapi.yaml:3]: Change title to our project name, but preserve `OpenAPI 3.0`

File: openapi.yaml [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+          description: nonogram id to delete
+          required: true
+          schema:
+            type: string
+            format: UUIDv4
+      responses:
+        '200':
+          description: successful operation
+        '400':
+          description: Invalid pet value
```
Comment (Frendzlu) [inline openapi.yaml:168]: change to `invalid id`

**Review by Frendzlu — CHANGES_REQUESTED** [review]

File: nsc_start.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (jFlamer) [reply nsc_start.py:3]: Alright, I'll go with homeData then

File: openapi.yaml (hunk 5 above)
Comment (jFlamer) [reply openapi.yaml:6]: OK, I'll make it point to repository

**Review by Frendzlu — APPROVED** [review]
lgtm

**Review by qualv13 — APPROVED** [review]

**Pull request merged**